Exercise_2.cpp

```diff
@@ -19,21 +19,45 @@ StackNode* newNode(int data)
 int isEmpty(StackNode* root) 
 { 
     //Your code here 
+    return !root;
 } 
   
 void push(StackNode** root, int data) 

```

Comment:
Please include:
```
// Time Complexity :
// Space Complexity :
// Did this code successfully run on Leetcode :
// Any problem you faced while coding this :


// Your code here along with comments explaining your approach
```

Reply:
@jigar99 This is a general template. Please follow this for the rest of the PR submissions after `PreCourse`. For this problem, time and space complexity was required for each method.

Reply:
@shrivatsav Where I will find leetcode link to submit?